pkg/ephemeral-disk/ephemeral-disk_test.go

```diff
@@ -50,38 +46,17 @@ var _ = Describe("ContainerDisk", func() {
 		if err != nil {
 			return err
 		}
-		return f.Close()
```

Comment:
I think you should do f.close as a defer function to make sure its done even if next checks return error.